libs/blocks/article-header/article-header.js

```diff
@@ -115,37 +144,47 @@ async function buildSharing() {
     link.addEventListener('click', openPopup);
   });
   const copyButton = sharing.querySelector('#copy-to-clipboard');
-  copyButton.addEventListener('click', async () => {
-    const copyText = await replaceKey('copied-to-clipboard', getConfig());
-    await copyToClipboard(copyButton, copyText);
-  });
+  copyButton.addEventListener('click', () => copyToClipboard(copyButton, copyText));
 
   return sharing;
 }
 
-async function validateDate(date) {
+function validateDate(date) {
   const { env } = getConfig();
   if (env?.name === 'prod') return;
   if (date && !/^[0-1]\d{1}-[0-3]\d{1}-[2]\d{3}$/.test(date.textContent.trim())) {
     // match publication date to MM-DD-YYYY format
     date.classList.add('article-date-invalid');
-    date.setAttribute('title', await replaceKey('invalid-date', getConfig()));
+    date.setAttribute('title', 'Invalid Date Format');
```

Comment:
Looks like this is only used by authoring, could we remove the comment and make the title more descriptive?